app/code/Magento/Customer/Model/Url.php

```diff
@@ -237,7 +237,7 @@ public function getForgotPasswordUrl()
      */
     public function getEmailConfirmationUrl($email = null)
     {
-        return $this->urlBuilder->getUrl('customer/account/confirmation', ['email' => $email]);
+        return $this->urlBuilder->getUrl('customer/account/confirmation', array('_query' => ['email' => $email]));
```

Comment:
Please use short array syntax (`[]` instead of `array()`)

Reply:
@ishakhsuvarov thank's. Adapted and pushed.